en_ewt-ud-dev.conllu

```diff
@@ -1751,8 +1751,8 @@
 8	to	to	PART	TO	_	9	mark	9:mark	_
 9	prove	prove	VERB	VB	VerbForm=Inf	7	xcomp	7:xcomp	_
 10	just	just	ADV	RB	_	12	advmod	12:advmod	_
-11	how	how	ADV	WRB	PronType=Int	12	advmod	12:advmod	_
-12	fanatic	fanatic	ADJ	JJ	Degree=Pos	9	ccomp	9:ccomp	_
+11	how	how	ADV	WRB	PronType=Int	12	advmod	12:advmod	CxnElt=12:Interrogative-WHInfo-Indirect.WHWord
+12	fanatic	fanatic	ADJ	JJ	Degree=Pos	9	ccomp	9:ccomp	Cxn=Interrogative-WHInfo-Indirect|CxnElt=12:Interrogative-WHInfo-Indirect.Clause
```

Comment:
I think this is actually exclamative not interrogative (but it would require manual disambiguation).